tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -232,6 +236,705 @@ def test_bobby_profiler_user_workflow_multi_batch_row_count_range_rule_and_colum
+    }
+
+    result = validator.expect_column_values_to_be_between(
+        column=column_name,
+        min_value=0.0,
+        result_format="SUMMARY",
+        include_config=True,
+        auto=True,
+        profiler_config=custom_profiler_config,
+    )
+    assert not result.success
+    assert result.expectation_config["kwargs"] == {
+        "column": "fare_amount",
+        "min_value": 0.0,
+        "strict_min": False,
+        "mostly": 1.0,
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "auto": True,
+        "profiler_config": custom_profiler_config,
+        "batch_id": "90bb41c1fbd7c71c05dbc8695320af71",
+    }
+
+    result = validator.expect_column_values_to_be_between(
+        column=column_name,
+        min_value=0.0,
+        mostly=8.75e-1,
+        result_format="SUMMARY",
+        include_config=True,
+        auto=True,
+        profiler_config=custom_profiler_config,
+    )
+    assert result.success
+    assert result.expectation_config["kwargs"] == {
+        "column": "fare_amount",
+        "min_value": 0.0,
+        "strict_min": False,
+        "mostly": 8.75e-1,
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "auto": True,
+        "profiler_config": custom_profiler_config,
+        "batch_id": "90bb41c1fbd7c71c05dbc8695320af71",
+    }
+
+    with pytest.raises(AssertionError) as e:
+        # noinspection PyUnusedLocal
+        result = validator.expect_column_values_to_be_between(
+            column=column_name,
+            mostly=1.0,
+            result_format="SUMMARY",
+            include_config=True,
+            auto=False,
+            profiler_config=custom_profiler_config,
+        )
+    assert "min_value and max_value cannot both be None" in str(e.value)
+
+    batch_request = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+        "data_connector_query": {
+            "index": 1,
+        },
+    }
+
+    validator = context.get_validator(
+        batch_request=BatchRequest(**batch_request),
+        expectation_suite_name=expectation_suite_name,
+    )
+    assert len(validator.batches) == 1
+
+    # Use one batch (at index "1"), loaded by Validator as active_batch, for DomainBuilder purposes.
+    domain_batch_request: dict = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+        "data_connector_query": {
+            "index": -1,
+        },
+    }
+
+    # Use all batches, except ("latest") active_batch, loaded by Validator, for estimating Expectation argument values.
+    parameter_builder_batch_request: dict = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+        "data_connector_query": {
+            "index": ":-1",
+        },
+    }
+
+    custom_profiler_config = {
```

Comment:
this should be a config type

Reply:
@donaldheppner -- same reason -- users may not want to go through the extra step of creating the config object -- we can give them both options.

Reply:
@donaldheppner This got done, based on our conversation.  Thanks!